docs/content/docs/connectors/datastream/jdbc.md

```diff
@@ -32,14 +32,6 @@ To use it, add the following dependency to your project (along with your JDBC dr
 
 {{< artifact flink-connector-jdbc withScalaVersion >}}
 
-A driver dependency is also required to connect to a specified database. Here are drivers currently supported:
-
-| Driver      |      Group Id      |      Artifact Id       |      JAR         |
-| :-----------| :------------------| :----------------------| :----------------|
-| MySQL       |       `mysql`      | `mysql-connector-java` | [Download](https://repo.maven.apache.org/maven2/mysql/mysql-connector-java/) |
-| PostgreSQL  |  `org.postgresql`  |      `postgresql`      | [Download](https://jdbc.postgresql.org/download.html) |
-| Derby       | `org.apache.derby` |        `derby`         | [Download](http://db.apache.org/derby/derby_downloads.html) |
-
```

Comment:
Because "Here are drivers currently supported" sounds like others are not supported (and maintaining the list of **all** the supported drivers seems unfeasable).

Reply:
I agree. But could we retain the first sentence and encourage looking for the JDBC dep?

Reply:
If that's the case, I would suggest still keeping this list, and interpret it as an

example JDBC dep list. This will give some examples and user-friendly for first-time users. WDYT?

Reply:
I think for production, users would still go to the vendor website to check the driver version and distribution they should use (regardless whether first-time users or not); or use some previously used driver.

One issue with these links is that we have to rely on users feedback to see if these link are still correct.
They are also a bit inconsistent: for Postgres it's Postgres website and for MySQL it's maven repo.

So IMO we shouldn't keep them.

Reply:
I do not have a strong opinion on this. 

- Those things (listed) do not change very often.
- The point here is to give some guidances. It will provide some hints for users and for other not listed dbs what they are looking for.
-  JDBC SQL Connector provides similar guides:
https://ci.apache.org/projects/flink/flink-docs-release-1.13/docs/connectors/table/jdbc/

Reply:
Ok, I'll remove it then, thanks!